pytypes/typechecker.py

```diff
@@ -425,8 +429,8 @@ def override(func, auto = False):
                     base_classes[i] = derived_class_locals[base_class]
                 elif base_class in derived_class_globals:
                     base_classes[i] = derived_class_globals[base_class]
-                elif base_class in types.__builtins__:
-                    base_classes[i] = types.__builtins__[base_class]
+                elif hasattr(types, base_class):
+                    base_classes[i] = getattr(types, base_class)
```

Comment:
I suspect this change breaks the tests. There can be a subtle difference between these versions.